boot/debug.go

```diff
@@ -112,6 +108,15 @@ func DebugSetBootVars(dir string, recoveryBootloader bool, varEqVal []string) er
 	case InitramfsUbuntuSeedDir:
 		opts.Role = bootloader.RoleRecovery
 	}
+	if recoveryBootloader {
+		// UC20 recovery bootloader
+		opts.Role = bootloader.RoleRecovery
+		if !opts.NoSlashBoot {
+			// no root dir was provided, use the default one for a
+			// recovery bootloader
+			dir = InitramfsUbuntuSeedDir
```

Comment:
we need to set NoSlashBoot to true as well, no?

Reply:
As discussed on IRC, we do not need to set NoSlashBoot as it is effectively ignored when using RoleRecovery.